MainRepository/Program.cs

```diff
@@ -1,7 +1,31 @@
-﻿internal class Program
-{
-    private static void Main(string[] args)
-    {
-        Console.WriteLine("Hello, World!");
-    }
-}
\ No newline at end of file
+﻿// Program to convert the decimal number to binary and hexadecimal number.
+Console.Write ("Enter the number: ");
+int number, num, num1;

```

Comment:
As you need number in two cases. Have int input and copy input to number whenever you need.